core/debugger/engine_debugger.h

```diff
@@ -92,6 +94,8 @@ class EngineDebugger {
 
 	uint32_t poll_every = 0;
 
+	Ref<Shortcut> stop_shortcut;
```

Comment:
This doesn't seem to work for GCC/Clang, I guess it might be missing `Ref<T>` definition?

Reply:
Now I'm getting the same error as in https://github.com/godotengine/godot/pull/80330 🤔